{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "name": "BMMB554_Project_Yi(yzl452).ipynb",
      "version": "0.3.2",
      "provenance": [],
      "collapsed_sections": [],
      "toc_visible": true,
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/yzl452/BMMB554/blob/master/BMMB554_Project_Yi(yzl452).ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "metadata": {
        "id": "eJVFegjeBNYx",
        "colab_type": "text"
      },
      "cell_type": "markdown",
      "source": [
        "# Using Methyltransferase Targeting Assay to study chromosome interaction at genome-wide scale"
      ]
    },
    {
      "metadata": {
        "id": "m4qSeLx9BctQ",
        "colab_type": "text"
      },
      "cell_type": "markdown",
      "source": [
        "## Abstract"
      ]
    },
    {
      "metadata": {
        "id": "By12DCK8hHB8",
        "colab_type": "text"
      },
      "cell_type": "markdown",
      "source": [
        "Chromosome interaction plays an important role in eukaryotic gene regulation and many methods have been developed to study the dynamic chromosome interaction, including 3C, 4C and Hi-C. However, these chemical based methods rely on a crosslink that immobilizes the dynamic chromosome interactions, it is still under debate regarding the bias introduced by chemical crosslink. And discrepancies between microscopy based chromosome structure and chemical based structure have been reported. To overcome these problems, here I describe a new tool to study chromosome interactions at a genome-wide scale based on a methyltransferase targeting assay. LacO sites were inserted into region of interest and a LacI-MCviPI (a GpC methyltransferase) fusion protein was expressed and recruited to LacO insertion site in budding yeast. A strain with mutant LacO sites was generated as a control. By searching for the hypermethylated regions in yeast strain with wild-type LacO sites that generated by the proximity to fusion protein target across the whole genome, the interactions between the hypermethylated regions and the LacO insertion site are captured. To simplify, Here I only focus on the interaction regions on chromosome III, which is where LacO sites were inserted."
      ]
    },
    {
      "metadata": {
        "id": "OLcQVfdfBfq_",
        "colab_type": "text"
      },
      "cell_type": "markdown",
      "source": [
        "## Introduction"
      ]
    },
    {
      "metadata": {
        "id": "YKkRpB_eBnwy",
        "colab_type": "text"
      },
      "cell_type": "markdown",
      "source": [
        "## Result"
      ]
    },
    {
      "metadata": {
        "id": "8nLGQxBMw7sj",
        "colab_type": "text"
      },
      "cell_type": "markdown",
      "source": [
        "### Pre-sequencing treatment"
      ]
    },
    {
      "metadata": {
        "id": "G_s7KRFpyDBQ",
        "colab_type": "text"
      },
      "cell_type": "markdown",
      "source": [
        "### Quality control"
      ]
    },
    {
      "metadata": {
        "id": "kr_FM05JyIPy",
        "colab_type": "text"
      },
      "cell_type": "markdown",
      "source": [
        "### Mapping"
      ]
    },
    {
      "metadata": {
        "id": "38Hg5UzUzOQW",
        "colab_type": "text"
      },
      "cell_type": "markdown",
      "source": [
        "### DMR Identification"
      ]
    },
    {
      "metadata": {
        "id": "HwfnO3WHBqfC",
        "colab_type": "text"
      },
      "cell_type": "markdown",
      "source": [
        "## Discussion"
      ]
    }
  ]
}